pkg/KubeArmorMacro/config/crd/bases/security.kubearmor.com_kubearmormacroes.yaml

```diff
@@ -0,0 +1,67 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: kubearmormacroes.security.kubearmor.com
+spec:
+  group: security.kubearmor.com
+  names:
+    kind: KubeArmorMacro
+    listKind: KubeArmorMacroList
+    plural: kubearmormacroes
```

Comment:
it seems that kubebuilder automatically added "es".
you can have sed -i "s/kubearmormacroes/kubearmormacros" XXXX in Makefile.